### DISCUSSION TITLE
Thinkpad Support Confirmed

### DISCUSSION THREAD

**Discussion by ZSHUser01** [tshakalekholoane/bat#23] [General]
Hi, 
I'm writing to you to let you know that I have tried your excellent program on my Lenovo Thinkpad x280 (the one with 8th gen Intel Core CPU) and I'm happy to let you know that it works perfectly. 

I'd suggest you to update the README in order to let other people know the program works with thinkpads too if you consider it the case obviously.

Keep up the great work

**Comment by tshakalekholoane** [comment]
That's great! Thank you. I will.
